### DISCUSSION TITLE
Kibana, Grafana and suricata missing.

### DISCUSSION THREAD

**Discussion by shapesh** [Security-Onion-Solutions/securityonion#1669] [Unsupported Versions] [answered]
After updating to 2.3.2 all containers went down. I had to manually put them up.
after running so-status some services, Grafana, Kibana and Suricata are missing.
How can I solve this?
![Capture](https://user-images.githubusercontent.com/11378515/97278577-fe6af100-184a-11eb-9f02-3fce8487bf4c.PNG)

**Comment by weslambert** [comment, marked as answer]
Is this still an issue?

**Reply by shapesh** [reply]
No the issue was solved.
